src/Workspaces/Core/Portable/Utilities/NullableHelpers/NamedTypeSymbolWithNullableAnnotation.cs

```diff
@@ -55,6 +55,11 @@ public INamedTypeSymbol Construct(params ITypeSymbol[] typeArguments)
                 return WrappedSymbol.Construct(typeArguments).WithNullability(Nullability);
             }
 
+            public INamedTypeSymbol Construct(ImmutableArray<ITypeSymbol> typeArguments, ImmutableArray<NullableAnnotation> typeArgumentNullableAnnotations)
+            {
+                return WrappedSymbol.Construct(typeArguments, typeArgumentNullableAnnotations);
```

Comment:
>; [](start = 94, length = 1)

.WithNullability(Nullability)

Reply:
Correct: should work like the method above.

Reply:
Good catch. Logged [https://github.com/dotnet/roslyn/issues/37279](https://github.com/dotnet/roslyn/issues/37279)

---
In reply to: [304076179](https://github.com/dotnet/roslyn/pull/37271#discussion_r304076179) [](ancestors = 304076179)